DC-SLES-amd-sev

```diff
@@ -0,0 +1,19 @@
+## ---------------------------- 
+## Doc Config File for SUSE Linux Enterprise AMD Secure Encrypted Virtualization
+## (AMD-SEV)
+## ----------------------------
+##
+## Basics
+MAIN="art_amd-sev.xml"
+ROOTID="art.amd-sev"
```

Comment:
Not sure if this makes sense, but... maybe you integrate your file into the `book_quickstarts.xml` file? Or is it too distinctive so it doesn't fit into this Quick Start Guide?

One benefit would be to simplify the linking to parts of the Virtualization Best Practices Guide.

Reply:
I have no opinion on this. @fsundermeyer, what do you think?

Reply:
For now it is a stand-alone guide. I expect it to end up in the virtualization guide some day anyway, so ATM I would not add it.